### DISCUSSION TITLE
英國法院宣判14歲少女Molly Russell死於「演算法推薦內容」

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#690] [General]
驗屍官：“Molly Rose Russell died from an act of self-harm while suffering from depression and the negative effects of online content.”不以自殺說明這位少女的死，而是歸咎於演算法。
社群媒體的推薦，或許是加劇一個人焦慮症的原因，但是卻從來沒有人拿出證據並且確切地宣稱「演算法推薦給我的內容就是我的焦慮症元凶」，而科技巨頭們也極力宣稱「個人化的內容，是基於使用者的習慣作為推薦的依據」。

這個時代裡每個使用者開啟Instagram、Youtube等軟體時，將會留下個人足跡，且全部都會被紀錄下來，變成一個個人模型，作為分析行為模式的數據來源。本來就很悲觀的人或許因為這樣的模型，社群媒體不斷推薦相關的內容或是更能激起他們暴力衝動的內容，因此造成無法挽回的後果。我覺得這些科技公司或是可以將這些較為悲觀的詞彙或內容進行特別管理或是限制，且要禁止推送更悲觀的內容給可能輕生者，利用AI進行這些限制應該不難吧，我認為這是這些科技公司該做的事，這樣才合乎道德與保護使用者的理念。
你們覺得演算法的推送究竟要如何才算是合乎道德的呢？

文章來源：https://www.thenewslens.com/article/178433

**Comment by 0212Jerry** [comment]
這確實是個難題，畢竟看到的東西也是由個人先前的選擇而來的，所以我認為應該要修正演算法，讓演算法不僅限於講更精準的內容推薦給使用者，而是進一步演算使用者所選擇的內容背後的傾向或意圖，並試圖增加或減少推送相關的內容。

**Comment by Valentinaa0214** [comment]
我認為不能單單只是怪罪在演算法上，造成用戶焦慮的原因可能包括推薦的內容本身、推薦的頻率、用戶目前的心理狀態等等。或許未來演算法的推薦內容要更人性化一些，把用戶的情緒也考慮進去之類的。

**Comment by Vincent20030314** [comment]
未來演算法如果可以透過觀看時間和頻率等等判斷是否繼續推薦的話可能可以減少這方面的問題。

**Comment by titan92601** [comment]
這其實是十分嚴重的問題。演算法是至今各大社交平台必備的功能，IG,YT,TikTok等等都有。然而有人因此沈迷於網路上，也產生不少心理問題和身體疾病。
